docs/about/privacy.md

```diff
@@ -0,0 +1,170 @@
+
+Only members of the CFDE staff, its approved contractors, and its approved vendors have access to the data collected by the Services. The CFDE may share information with its third-party vendors and service providers, to comply with legal obligations, to protect and defend our rights and property, and with your permission. The CFDE may share any information it receives with vendors and service providers it uses to help provide and improve the Services. For example, the CFDE may disclose information to the service providers that help host the Services or send its emails.
+
+The CFDE may allow access to data collected by the Services to send you information the CFDE thinks may be useful or relevant to you.
+
+The CFDE may access, preserve, and disclose collected information, if it believes doing so is required or appropriate to: comply with law enforcement requests and legal process, such as a court order or subpoena; respond to your requests; comply with the law; or protect your, our, or others’ rights, property, or safety.
+
+If the CFDE is involved in a transition of service to another provider, your information may be disclosed in connection with the negotiation of such transaction, and/or transferred as part of such a transaction as permitted by law and/or contract. The CFDE cannot control how the recipient entities may use or disclose such information.
+
+## Children’s Privacy
+
+The CFDE is committed to protecting the privacy of children. The Services are intended for adults and young adults ages 13 and over. It is not intended for or designed to attract persons under the age of 13. Registration by any person to be under the age of 13 is not permitted and no part of the Services are directed to children under 13. If the CFDE learns that a child under 13 has created an account, the CFDE will promptly take steps to delete such information and terminate the child’s account.
+
+## Other Sites
+
+Our privacy policies apply only to your use of the Services. The Services contain links and integrations to other sites, including sites that may indicate relationships or collaborations with the CFDE. At times, in support of technical integrations, you may be required to provide your email address and name to third parties to fully use and/or access all Services. The CFDE is not responsible for the privacy practices of such other sites. You should read and review the privacy policies of each site you visit to determine what information that site may be collecting about you. Below are the privacy policies for the third party websites & services that the Services currently integrate with:
+
+- Google: Users have the option to integrate their CFDE Portal login with Google to allow users to “login with Google.” The Services will access your basic profile information including your name and email address. You may visit Google’s privacy policy here: [https://policies.google.com/privacy](https://policies.google.com/privacy)
```

Comment:
I think we can take the text from the Google login and move it under Globus, since that's our login method and Globus provides the same OIDC user info.

Reply:
We will also remove the Google and ORCiD text.